src/core/qgsdataprovider.h

```diff
@@ -628,18 +629,18 @@ class CORE_EXPORT QgsDataProvider : public QObject
     //! Sets error message
     void setError( const QgsError &error ) { mError = error;}
 
-  private:
```

Comment:
This change doesn't seem required -- can you revert?

Reply:
I have fixed it by created `mReadFlags`